src/model/documentselection.js

```diff
@@ -750,14 +750,14 @@ class LiveSelection extends Selection {
 
 		const liveRange = LiveRange.createFromRange( range );
 
-		liveRange.on( 'change:range', ( evt, oldRange, operation ) => {
+		liveRange.on( 'change:range', ( evt, oldRange, deletionPosition ) => {
```

Comment:
It should be an object in case we need to add more properties in the future.

Reply:
👍